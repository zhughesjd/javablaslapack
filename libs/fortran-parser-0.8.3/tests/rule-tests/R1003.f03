

! Testing format item list. Ripped off from standard.
! Note that the list must have at least two elements.
! Also, should the second list count as four or six?
format (1pe12.4, i10)
format (i12, /, ' Dates: ', 2 (2i3, i5))
format (i15)
end

