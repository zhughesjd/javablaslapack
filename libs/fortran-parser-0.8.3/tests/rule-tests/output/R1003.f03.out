1101:begin
1003:list-begin
1003:list 2
1003:list-begin
1003:list-begin
1003:list 2
1003:list 6
1103
1101 false false false
